content/api-docs/entry/alert-group-settings.md

```diff
@@ -44,34 +44,34 @@ CustomPath: alert-group-settings
 | `id`                   | *string*        | the alert group setting ID                                                 |
 | `name`                 | *string*        | the name of alert group setting                                             |
 | `memo`                 | *string*        | [optional] notes related to the alert group setting                        |
-| `serviceScopes`        | *array[string]* | [optional] scope for the target service. an array of the service name                     |
-| `roleScopes`           | *array[string]* | [optional] scope for the target role. an array of the role fullname [*1](#role-fullname) |
-| `monitorScopes`        | *array[string]* | [optional] scope for the target monitor. an array of the monitor ID                      |
-| `notificationInterval` | *number*        | [optional] the time interval (in minutes) for  re-sending notifications                    |
+| `serviceScopes`        | *array[string]* | [optional] scope for the target service. an array of service names                     |
+| `roleScopes`           | *array[string]* | [optional] scope for the target role. an array of role fullnames [*1](#role-fullname) |
+| `monitorScopes`        | *array[string]* | [optional] scope for the target monitor. an array of monitor IDs                      |
+| `notificationInterval` | *number*        | [optional] the time interval (in minutes) for  resending notifications                    |
 
 <h4 id="role-fullname">*1 role fullname</h4>
 
-A role's full name is an array in the format of `<service name>:<role name>`.
+A role's fullname is an array in the format of `<service name>:<role name>`.
 
 <table class="eg-table">
   <tbody>
   <tr>
     <th>e.g.</th>
-    <td>if a Hatena-Bookmark service db-master role <code>Hatena-Bookmark:db-master</code></td>
+    <td>if you have <code>Hatena-Bookmark</code> as the service and <code>db-master</code> as the role, its role fullname is <code>Hatena-Bookmark:db-master</code></td>
     </tr>
   </tbody>
 </table>
 
-available strings are `/^[A-Za-z0-9][A-Za-z0-9_-]+$/`
+available strings for the role fullname must match the following regular expression: `/^[A-Za-z0-9][A-Za-z0-9_-]+$/`
```

Comment:
I guess if you are an engineer you would know that `/^[A-Za-z0-9][A-Za-z0-9_-]+$/` is a regular expression, but I felt it would be more kind to explain it like this.